src/solver/modeler/ortoolsImpl/include/antares/solver/modeler/ortoolsImpl/linearProblem.h

```diff
@@ -67,8 +67,6 @@ class OrtoolsLinearProblem: public Api::ILinearProblem
     OrtoolsMipSolution* solve(bool verboseSolver) override;
 
     double infinity() const override;
-
-protected:
```

Comment:
Should stay protected to keep implementation details away in `src/solver/modeler/main.cpp`

Reply:
Yes, the API currently lacks the ability to loop over variables ! All we have is
```
IMipVariable* getVariable(const std::string& name)
```
which is kind of useless when we want to loop over all variables (unless we already know their name, even then it's not very convenient).